rootfs/usr/local/bin/docs

```diff
@@ -29,9 +29,12 @@ tasks:
       mkdir -p ${man_pages}
       for md in $(find /usr/share/docs -type f -name '*.md'); do
         man=${man_pages}/$(basename $md .md).$section
-        # Verbatim text (i.e. between backticks, `…`) doesn’t show in any particular way
-        sed 's/`/**/g' < $md | sed -E 's/^#(#+ )/\1/g' | \
-          pandoc -s -f markdown -t man | sed 's/\\\[en\]/-/' > $man
+        # Verbatim text (i.e. between backticks, `…`) doesn’t show in any particular way, so we bold it
+        # by using "**", but we need to preserve code fence triple backticks "```".
```

Comment:
```suggestion
        # by using "**", but we need to preserve code hence triple backticks "```".
```

Reply:
No, the triple backticks is called a "code fence".